web-crypto/derive-key/hkdf.js

```diff
@@ -26,7 +26,7 @@
     return window.crypto.subtle.deriveKey(
       {
         name: "HKDF",
-        salt: salt,
+        salt,
```

Comment:
This was changed in https://github.com/mdn/content/commit/2c0f972d873ea2db5163dbcb12987847124751ad#diff-12e29f1ce3a1b155e75b708ef543437aaabe920f24e5cb4697e2187b11112b09